compiler/modules/nback/scaffold.bal

```diff
@@ -239,6 +240,17 @@ class Scaffold {
         }
     }
 
+    function buildRuntimeFunctionCall(llvm:Builder builder, RuntimeFunction rf, llvm:Value[] args) returns llvm:Value? {
+        return self.buildCall(builder, self.getRuntimeFunctionDecl(rf), args);
+    }
+
+    function buildCall(llvm:Builder builder, llvm:Function fn, llvm:Value[] args) returns llvm:Value? {
```

Comment:
Don't make this a method.

Reply:
fixed (I assume this is to move the function out of scaffold)